modules/framework/galasa-parent/galasa-boot/src/main/java/dev/galasa/boot/Launcher.java

```diff
@@ -172,7 +171,7 @@ protected void launch(String[] args) throws LauncherException, InterruptedExcept
                 felixFramework.runTest(bootstrapProperties, overridesProperties);
             } else if (isResourceManagement) {
                 logger.debug("Resource Management");
-                MonitorConfiguration monitorConfig = getResourceMonitorConfigFromEnvironment(env);
+                MonitorConfiguration monitorConfig = new MonitorConfiguration(env);
```

Comment:
Should this be called ResourceManagementConfiguration ?

Reply:
Renamed in https://github.com/galasa-dev/galasa/pull/236/commits/a9ef197a47c1c7da97f4c1566f80ca24bce91184